### PR TITLE
fix linters

### PR DESCRIPTION
**Pull request by ijeomaodinko**
Project requirements
Create a new Rails app called 'hello-rails-react'.
Use the commend rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
Verify that the jsbundling-rails gem is added to your Gemfile.
Initialize your project with Git.
Install needed node dependencies as described in the previous lesson's tutorial.
Create a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
Update webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
Make sure that your project has a Postgres database set up.
Create a table for storing your messages and create 5 different greetings.
Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
Create a static view that will be the root of your app.

### CODE REVIEW

File: app/controllers/api/v1/greetings_controller.rb [hunk 1]
```diff
@@ -1,12 +1,14 @@
+module Api
+  module V1
+    class GreetingsController < ApplicationController
```
Comment (brytebee) [inline app/controllers/api/v1/greetings_controller.rb:3]: Kindly look into the following errors I got trying to run your project. These errors came after running the rails server and opening the port on the browser.

- This is from the console.
image

- This is from the page.
image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ijeomaodinko👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- The README looks professional.✔️
- No linter errors.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/api/v1/greetings_controller.rb (hunk 1 above)
Comment (mahbubul14) [reply app/controllers/api/v1/greetings_controller.rb:3]: > Kindly look into the following errors I got trying to run your project. These errors came after running the rails server and opening the port on the browser.
> 
> * This is from the console.
>   image
> * This is from the page.
>   image

- [ ] Kindly fix the issue addressed by the previous reviewer. I got the same issue while checking. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @ijeomaodinko,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by ijeomaodinko** [comment]
Hi, but this I have fixed it, and am not getting it in my system.

**Pull request closed without merging**